src/3.arrays.js

```diff
@@ -6,50 +6,50 @@
  * indexoff(arr, 3); should return the index 2
  */
 function indexOf(arr, item) {
-  return ;
+  return arr.indexOf(item);
 }
 
 // Sum: you should be able to sum the items of an array
 // Example: arr = [1,2,3]   =>  6
 function sum(arr) {
-  return ;
+  return arr.reduce((a,b) => a+b,0);
 }
 
 // Filter: Should filter out all instances of a value from an array
 // Example: filtering 2 out of [1,2,3,5,6,2,4,2] should return [1,3,5,6,4]
 function filterOut(arr, item) {
-  return ;
+  return arr.filter(i => i !== item);
 }
 
 // Append: you should be able to add an item to the end of an array
 function append(arr, item) {
-
-  return ;
+  arr.push(item);
+  return arr;
 }
 
 // Truncate: you should be able to remove the last item of an array
 // hint: pop, slice, splice would all work. Pay attention to the return value in the docs
 function truncate(arr) {
-  return ;
+  return arr.splice(0,2);
```

Comment:
Not what we intended, but it does show flaws in the tests for this. 

Using `splice` the answer we were looking for was:

```js
arr.splice(arr.length - 1, 1);
return arr
```

Using `pop`:

```js
arr.pop()
return arr
```
Using `slice`:

```js
return arr.slice(0, arr.length - 1);
```